grpc-ballerina/tests/43_nested_record_with_streams_pb.bal

```diff
@@ -0,0 +1,287 @@
+        check self.grpcClient.initStub(self, ROOT_DESCRIPTOR_43, getDescriptorMap43());
+    }
+
+    isolated remote function nestedMsgUnary(string|ContextString req) returns (NestedMsg|Error) {
+        map<string|string[]> headers = {};
+        string message;
+        if (req is ContextString) {
+            message = req.content;
+            headers = req.headers;
+        } else {
+            message = req;
+        }
+        var payload = check self.grpcClient->executeSimpleRPC("NestedMsgService/nestedMsgUnary", message, headers);
+        [anydata, map<string|string[]>] [result, _] = payload;
+        return <NestedMsg>result;
+    }
+
+    isolated remote function nestedMsgUnaryContext(string|ContextString req) returns (ContextNestedMsg|Error) {
+        map<string|string[]> headers = {};
+        string message;
+        if (req is ContextString) {
+            message = req.content;
+            headers = req.headers;
+        } else {
+            message = req;
+        }
+        var payload = check self.grpcClient->executeSimpleRPC("NestedMsgService/nestedMsgUnary", message, headers);
+        [anydata, map<string|string[]>] [result, respHeaders] = payload;
+        return {content: <NestedMsg>result, headers: respHeaders};
+    }
+
+    isolated remote function nestedMsgClientStreaming() returns (NestedMsgClientStreamingStreamingClient|Error) {
+        StreamingClient sClient = check self.grpcClient->executeClientStreaming("NestedMsgService/nestedMsgClientStreaming");
+        return new NestedMsgClientStreamingStreamingClient(sClient);
+    }
+
+    isolated remote function nestedMsgServerStreaming(string req) returns stream<NestedMsg, Error?>|Error {
+        var payload = check self.grpcClient->executeServerStreaming("NestedMsgService/nestedMsgServerStreaming", req);
+        [stream<anydata, Error?>, map<string|string[]>] [result, _] = payload;
+        NestedMsgStream outputStream = new NestedMsgStream(result);
+        return new stream<NestedMsg, Error?>(outputStream);
+    }
+
+    isolated remote function nestedMsgServerStreamingContext(string req) returns ContextNestedMsgStream|Error {
+        var payload = check self.grpcClient->executeServerStreaming("NestedMsgService/nestedMsgServerStreaming", req);
+        [stream<anydata, Error?>, map<string|string[]>] [result, headers] = payload;
+        NestedMsgStream outputStream = new NestedMsgStream(result);
+        return {content: new stream<NestedMsg, Error?>(outputStream), headers: headers};
+    }
+
+    isolated remote function nestedMsgBidirectionalStreaming() returns (NestedMsgBidirectionalStreamingStreamingClient|Error) {
+        StreamingClient sClient = check self.grpcClient->executeBidirectionalStreaming("NestedMsgService/nestedMsgBidirectionalStreaming");
+        return new NestedMsgBidirectionalStreamingStreamingClient(sClient);
+    }
+}
+
+public client class NestedMsgClientStreamingStreamingClient {
```

Comment:
Seems `Streaming` is repeated in the class name. Should it be `NestedMsgClientStreamingClient` instead?